app/models/comparison/metric_type.rb

```diff
@@ -0,0 +1,30 @@
+# == Schema Information
+#
+# Table name: comparison_metric_types
+#
+#  created_at :datetime         not null
+#  fuel_type  :integer          not null
+#  id         :bigint(8)        not null, primary key
+#  key        :string           not null
+#  units      :integer          not null
+#  updated_at :datetime         not null
+#
+# Indexes
+#
+#  index_comparison_metric_types_on_key_and_fuel_type  (key,fuel_type) UNIQUE
+#
+class Comparison::MetricType < ApplicationRecord
+  self.table_name = 'comparison_metric_types'
+
+  extend Mobility
+  include EnumFuelType
+
+  enum units: [:float, :date, :percent, :relative_percent]
+  # are there more? [:£, :co2, :kwh, :time, :string, :kw]
```

Comment:
Yes, I'll send you the list of types that are currently declared, but don't necessarily have to include here

Reply:
Thank you @ldodds 👍